src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's block in the address book.
+ */
+
```

Comment:
You don't need to leave a blank line between a header comment and the class :) Same for `Street`, `Unit`, and `Postal Code`